test/d2l-search-results-count.html

```diff
@@ -0,0 +1,61 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+		<title>d2l-search-results-count test</title>
+		<script src="../../webcomponentsjs/webcomponents-loader.js"></script>
+		<script src="../../web-component-tester/browser.js"></script>
+		<script src="../../iron-test-helpers/mock-interactions.js"></script>
+		<script src="../../iron-test-helpers/test-helpers.js"></script>
+		<link rel="import" href="../all-imports.html">
+	</head>
+	<body>
+		<test-fixture id="basic">
+			<template>
+				<d2l-search-results-count range-start="1" range-end="20" total-count="22" query="Financial Planning"></d2l-search-results-count>
+			</template>
+		</test-fixture>
+		<test-fixture id="basic-no-query">
+			<template>
+				<d2l-search-results-count range-start="1" range-end="20" total-count="22"></d2l-search-results-count>
+			</template>
+		</test-fixture>
+		<script>
+			describe('<d2l-search-results-count>', function() {
+				describe('basic', function() {
+					let basicFixture;
+
+					beforeEach(function(done) {
+						basicFixture = fixture('basic');
+						Polymer.RenderStatus.afterNextRender(basicFixture, function() {
```

Comment:
You can pass in `done` if you don't need to do any other initialization:
```suggestion
						Polymer.RenderStatus.afterNextRender(basicFixture, done);
```